### DISCUSSION TITLE
How do I get relative auto-imports

### DISCUSSION THREAD

**Discussion by bryanforbes** [microsoft/pyright#2379] [Q&A] [answered]
As I'm developing a package, I always want to use relative imports within that package. For instance, if I have the following modules:

**foo/common.py**
```python
class Base:
    ...
```

**foo/one.py**
```python
from .common import Base

class One(Base):
    ...
```

When I get suggestions from pyright via the LSP while developing these modules, it always gives the the suggestions as absolute imports. As an example, let's say I was going to develop `foo/two.py` and I start it out with the following:

```python
class Two(Base
```

At this point, the LSP will give me an auto-import of `from foo.common import Base`, and then I have to go and edit that import to be `from .common import Base`. Is there a configuration option I'm missing to enable relative imports within a package (excluding namespace packages, of course)?

**Comment by jakebailey** [comment, marked as answer]
We're tracking this on https://github.com/microsoft/pylance-release/issues/607; we currently don't have any support for auto-imports that are relative. I'd vote/discuss there if you're interested in that.

**Reply by bryanforbes** [reply]
Thank you!
